packages/unity-interface/dcl.ts

```diff
@@ -164,29 +165,34 @@ export async function startUnitySceneWorkers() {
   store.dispatch(signalParcelLoadingStarted())
 }
 
-export async function loadPreviewScene(ws?: string) {
+export async function getPreviewSceneId(): Promise<{ sceneId: string | null; sceneBase: string }> {
   const result = await fetch('/scene.json?nocache=' + Math.random())
 
   if (result.ok) {
     const scene = (await result.json()) as SceneJsonData
 
     const [sceneId] = await fetchSceneIds([scene.scene.base])
+    return { sceneId, sceneBase: scene.scene.base }
+  } else {
+    throw new Error('Could not load scene.json')
+  }
+}
 
-    if (sceneId) {
-      await reloadScene(sceneId)
-    } else {
-      console.log(`Unable to load sceneId of ${scene.scene.base}`)
-      debugger
-    }
-
-    // let transport: undefined | ScriptingTransport = undefined
+export async function loadPreviewScene(ws?: string) {
+  const { sceneId, sceneBase } = await getPreviewSceneId()
 
-    // if (ws) {
-    //   transport = WebSocketTransport(new WebSocket(ws, ['dcl-scene']))
-    // }
+  if (sceneId) {
+    await reloadScene(sceneId)
   } else {
-    throw new Error('Could not load scene.json')
+    console.log(`Unable to load sceneId of ${sceneBase}`)
+    debugger
```

Comment:
do we still need this debugger here ?

Reply:
`sceneId` being `null` here means that there is an issue somewhere else, so it might become handy to have it there :)